sccm-currentbranch/azuredeploy.json

```diff
@@ -0,0 +1,384 @@
+      "type": "Microsoft.Compute/virtualMachines",
+      "apiVersion": "2018-06-01",
+      "name": "[concat(parameters('prefix'),variables('vmrole')[copyIndex()])]",
+      "location": "[parameters('location')]",
+      "copy": {
+        "name": "vmcopy",
+        "count": 3
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Network/networkInterfaces/', toLower(parameters('prefix')), '-ni-',variables('vmrole')[copyIndex()])]"
+      ],
+      "properties": {
+        "osProfile": {
+          "computerName": "[concat(parameters('prefix'),variables('vmrole')[copyIndex()])]",
+          "adminUsername": "[parameters('adminUsername')]",
+          "adminPassword": "[parameters('adminPassword')]",
+          "windowsConfiguration": {
+            "provisionVmAgent": "true"
+          }
+        },
+        "hardwareProfile": {
+          "vmSize": "[variables('generalSettings').virtualMachinessize]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "MicrosoftWindowsServer",
+            "offer": "WindowsServer",
+            "sku": "2016-Datacenter",
+            "version": "latest"
+          },
+          "osDisk": {
+            "osType": "Windows",
+            "name": "[concat(parameters('prefix'),variables('vmrole')[copyIndex()],'-OsDisk')]",
+            "createOption": "FromImage",
+            "caching": "ReadWrite",
+            "managedDisk": {
+              "storageAccountType": "[variables('generalSettings').diskstorageAccountType]"
+            },
+            "diskSizeGB": 150
+          },
+          "dataDisks": []
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces', concat(toLower(parameters('prefix')), '-ni-',variables('vmrole')[copyIndex()]))]"
+            }
+          ]
+        },
+        "diagnosticsProfile": {
+          "bootDiagnostics": {
+            "enabled": false
+          }
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "apiVersion": "2018-06-01",
+      "name": "[concat(parameters('prefix'),variables('vmrole')[copyIndex()],'/WorkFlow')]",
+      "location": "[parameters('location')]",
+      "copy": {
+        "name": "extensioncopy",
+        "count": 3
+      },
+      "dependsOn": [
+        "[concat('Microsoft.Compute/virtualMachines/',parameters('prefix'),variables('vmrole')[copyIndex()])]",
+        "[resourceId('Microsoft.Storage/storageAccounts/', variables('storageAccountName'))]"
+      ],
+      "properties": {
+        "type": "CustomScriptExtension",
+        "publisher": "Microsoft.Compute",
+        "typeHandlerVersion": "1.4",
+        "autoUpgradeMinorVersion": true,
+        "settings": {
+          "fileUris": [
+            "[concat(parameters('_artifactsLocation'),'/scripts/WorkFlow-',variables('vmrole')[copyIndex()],'.ps1',parameters('_artifactsLocationSasToken'))]"
```

Comment:
remove leading / from /scripts and use the uri function

Reply:
Also, we put all related scripts in the fileUris list.

Reply:
Correct. Also put all related scripts in the fileUris list.